tests/php/Centreon/Domain/HostConfiguration/Model/HostTemplateTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * Copyright 2005 - 2020 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+declare(strict_types=1);
+
+namespace Tests\Centreon\Domain\HostConfiguration\Model;
+
+use Centreon\Domain\HostConfiguration\Model\HostTemplate;
+use Centreon\Domain\Media\Model\Image;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * This class is designed to test certain methods of the HostTemplate entity, especially those with exceptions.
+ *
+ * @package Tests\Centreon\Domain\HostConfiguration\Model
+ */
+class HostTemplateTest extends TestCase
+{
+    public function testBadNotificationOptionException(): void
+    {
+        $optionsValueToTest = 32;
+        $this->expectException(\InvalidArgumentException::class);
+        $this->expectExceptionMessage(sprintf('Invalid notification option (%d)', $optionsValueToTest));
+        $hostTemplate = new HostTemplate();
+        $hostTemplate->setNotificationOptions($optionsValueToTest);
```

Comment:
according to https://github.com/centreon/centreon/pull/9239/files?file-filters%5B%5D=.php#r530157460

here we could do
$hostTemplate = new HostTemplateTestStub::createEntityWithInvalidNotificationOptions()